java/ql/lib/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -265,6 +265,24 @@ private predicate sinkModelCsv(string row) {
       "java.net;URLClassLoader;false;URLClassLoader;(String,URL[],ClassLoader);;Argument[1];open-url",
       "java.net;URLClassLoader;false;URLClassLoader;(String,URL[],ClassLoader,URLStreamHandlerFactory);;Argument[1];open-url",
       "java.net;URLClassLoader;false;newInstance;;;Argument[0];open-url",
+      //JDBC URL
```

Comment:
These should be moved to qll files relevant to their particular framework, and use an extension class, like

```
private class ExampleClassName extends SinkModelCsv {
  override predicate row(string row) {
    row =
      [
         ... row data here ...
      ]
  }
}
```

For example, in Jdbc.qll you'll see `private class SqlSinkCsv extends SinkModelCsv {`

Note that whenever a qll defines models this way you should add an import for it in `module Frameworks` on line 79 of this file.